R/drive_trash.R

```diff
@@ -0,0 +1,111 @@
+#' Trash file on Google Drive.
```

Comment:
I just did a massive removal of "Google" from "Google Drive" in help files, because it comes up so much and I think it's basically implied at this point. We will spell it out in some key places, but otherwise let's just say "Drive".